client/src/components/payment/payment-summary.vue

```diff
@@ -1,41 +1,41 @@
 <template>
-  <v-card class='mb-5' v-if="summary">
-    <header class='font-weight-bold px-3 py-3'>
-      <slot name='header'>
-        <span>
-          <small>{{receipt.receiptDate}}</small>
-        </span>
-        <span style="float: right" v-if="receipt">
-          <small><small>Receipt No.</small></small> {{receipt.receiptNumber}}
-        </span>
-      </slot>
+  <v-card v-if="summary" class="payment-summary">
+
+    <header class="font-weight-bold pa-3">
+      <span>
+        <small>{{receipt.receiptDate}}</small>
+      </span>
+      <span style="float: right" v-if="receipt">
+        <small><small>Receipt No.</small></small> {{receipt.receiptNumber}}
+      </span>
     </header>
 
-    <div v-show='fetchError'>
-      <v-alert color='error' icon='warning' outlined>{{fetchError}}</v-alert>
+    <div v-show="fetchError">
+      <v-alert color="error" icon="mdi-alert" outlined>{{fetchError}}</v-alert>
     </div>
 
-    <ul class='fee-list' v-show='!fetchError'>
-      <li class='container fee-list__item' v-if="receipt">
-        <div class='fee-list__item-name'>Amount</div>
-        <div class='fee-list__item-value'>${{receipt.receiptAmount.toFixed(2)}} CAD</div>
+    <ul class="fee-list" v-show="!fetchError">
+      <li class="container fee-list__item" v-if="receipt">
+        <div class="fee-list__item-name">Amount</div>
+        <div class="fee-list__item-value">${{receipt.receiptAmount.toFixed(2)}} CAD</div>
       </li>
-      <li class='container fee-list__item' v-if="summary">
-        <div class='fee-list__item-name'>Status</div>
-        <div class='fee-list__item-value'>{{summary.statusCode}}</div>
+      <li class="container fee-list__item" v-if="summary">
+        <div class="fee-list__item-name">Status</div>
+        <div class="fee-list__item-value">{{summary.statusCode}}</div>
       </li>
-      <li class='container fee-list__item'>
-        <div class='fee-list__item-name'>Receipt</div>
-        <div class='fee-list__item-value'>
-          <v-btn @click='downloadReceipt' id='download-receipt-btn' class='primary' text small>Download PDF</v-btn>
+      <li class="container fee-list__item">
+        <div class="fee-list__item-name">Receipt</div>
+        <div class="fee-list__item-value">
+          <v-btn text small class="primary download-receipt-btn"
+            :loading="loading" @click="downloadReceipt()">Download PDF</v-btn>
```

Comment:
I added "loading" state here so it looks like it's doing something while the receipt is downloading.